ci/pipelines/control-plane.yml

```
@@ -62,3 +80,55 @@ jobs:
         text: ':green-circle: Control plane restarted - <https://pay-cd.deploy.payments.service.gov.uk/teams/$BUILD_TEAM_NAME/pipelines/$BUILD_PIPELINE_NAME/jobs/$BUILD_JOB_NAME/builds/$BUILD_NAME|Concourse build #$BUILD_NAME>'
         icon_emoji: ":concourse:"
         username: pay-concourse
+
+  - name: deploy-control-plane
+    serial: true
+    plan:
+    - get: pay-control-plane-src
+      trigger: true
+    - task: build-control-plane-image
+      privileged: true
+      output_mapping:
+        image: pay-control-plane-image
+      config:
+        platform: linux
+        image_resource:
+          type: registry-image
+          source:
+            repository: concourse/oci-build-task
+        inputs:
+          - name: pay-control-plane-src
+            path: .
+        outputs:
+          - name: image
+        run:
+          path: build
+    - put: pay-control-plane-latest
+      params:
+        image: pay-control-plane-image/image.tar
```

```suggestion
        image: pay-control-plane-image/image.tar
        additional_tags: pay-control-plane-src/.git/ref
```

In defence of this: pay-control-plane is on a knife edge barely working and only just buildable.....so.....in order to not lose the older images what do you think to adding the git sha as an additional tag so we get a tag of the git commit sha as well meaning as the latest tag moves dockerhub doesn't discard the older versions of the image?

---

Absolutely - that was on my mind earlier and I completely forgot about it. Thanks!

---

Wicked, that worked a treat :)